src/fragments/Dropdown.react.js

```diff
@@ -97,6 +98,7 @@ export default class Dropdown extends Component {
                     onInputChange={search_value => setProps({search_value})}
                     backspaceRemoves={clearable}
                     deleteRemoves={clearable}
+                    inputProps={{...inputProps, autoComplete: 'off'}}
```

Comment:
We're not making an `inputProps` prop anymore, right?

Reply:
inputProps is not a dash prop -  it's a prop in the react component.  In order to update autoComplete, it looked like it needed to be wrapped in this way.  (it works :-))

Reply:
Also I'm not sure why the tests failed.  I didn't make any changes to:
test_grva004_graph_prepend_trace[False]
test_grva005_graph_extend_trace[False]

Reply:
Right, it's a prop of `ReactDropdown`, but what I meant was:
```suggestion
                    inputProps={{autoComplete: 'off'}}
```
ie you're pulling it in [up above](https://github.com/plotly/dash-core-components/pull/923/files#diff-6d73123aa77cd179dde9336ee11b3c81ba48c5589b504e8bb9e885a806a05473R57) as if it were a dash prop.

Reply:
Not sure what's going on with those tests, but whenever a test fails in only one of the Python versions the first guess is it's just flaky. We'll see what happens when we try it again.

Reply:
Oh, I see.  I'll make the change.